twistGain:
  drone0: [0.100, 0.000,  0.000,  0.000,  0.000,  0.000,
            0.000, 0.100, 0.000,  0.000,  0.000,  0.000,
            0.000, 0.000,  0.100, 0.000,  0.000,  0.000,
            0.000, 0.000,  0.000,  0.100, 0.000,  0.000,
            0.000, 0.000,  0.000,  0.000,  0.100, 0.000,
            0.000, 0.000,  0.000,  0.000,  0.000,  0.100]
  drone1: [0.100, 0.000,  0.000,  0.000,  0.000,  0.000,
            0.000, 0.100, 0.000,  0.000,  0.000,  0.000,
            0.000, 0.000,  0.100, 0.000,  0.000,  0.000,
            0.000, 0.000,  0.000,  0.100, 0.000,  0.000,
            0.000, 0.000,  0.000,  0.000,  0.100, 0.000,
            0.000, 0.000,  0.000,  0.000,  0.000,  0.100]
  drone2: [-1.000, 0.000,  0.000,  0.000,  0.000,  0.000,
            0.000, -1.000, 0.000,  0.000,  0.000,  0.000,
            0.000, 0.000,  -1.000, 0.000,  0.000,  0.000,
            0.000, 0.000,  0.000,  -1.000, 0.000,  0.000,
            0.000, 0.000,  0.000,  0.000,  -1.000, 0.000,
            0.000, 0.000,  0.000,  0.000,  0.000,  -1.000]
